domain/asset.go

```diff
@@ -0,0 +1,17 @@
+package domain
+
+import (
+	"time"
+
+	"github.com/docker/distribution/uuid"
+)
+
+type Asset struct {
+	AssetID        uuid.UUID              `db:"id"`
+	AssetName      string                 `db:"name"`
+	Status         string                 `db:"status"`
+	Category       string                 `db:"category"`
+	PurchaseAt     time.Time              `db:"purchase_date"`
+	PurchaseCost   float64                `db:"initial_cost"`
```

Comment:
can you correct the db keys, 

here is the migration file:
https://github.com/vkhichar/asset-management/blob/master/migrations/1000_create_assets_up.sql